include/deal.II/matrix_free/operators.h

```diff
@@ -958,7 +958,7 @@ namespace MatrixFreeOperators
                     const LinearAlgebra::distributed::Vector<Number> &src,
                     const std::pair<unsigned int,unsigned int>  &cell_range) const
   {
-    FEEvaluation<dim, fe_degree, fe_degree+1, n_components, Number> phi(data);
+    FEEvaluation<dim, fe_degree, fe_degree+2, n_components, Number> phi(data);
```

Comment:
i suppose you needed this to get extra precision in case of non-Q1 mapping to make unit tests pass?